source/PlatformerHero.hx

```diff
@@ -0,0 +1,63 @@
+package;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+
+/**
+ * ...
+ * @author Cory Jackson
+ */
+class PlatformerHero extends FlxSprite 
```

Comment:
More of a general comment for the PlatformerHero class - the movement of this character was very similar to my movement example. I think I said so in class, but this doesn't particularly bother me as long as you understand how the code is working. 

That said, in my opinion the `PlatformerHero` feels pretty bad to move. The example I put together also feels bad, but mine was primarily made with the intent of simple and easy to follow. You're don't really have to share my concern in that way, and I'd be interested in seeing how you would write movement that feels good to you.